substrate/frame/identity/src/lib.rs

```diff
@@ -42,15 +42,25 @@
 //!
 //! ### Usernames
 //!
-//! The pallet provides functionality for username authorities to issue usernames. When an account
-//! receives a username, they get a default instance of `IdentityInfo`. Usernames also serve as a
-//! reverse lookup from username to account.
+//! The pallet provides functionality for username authorities to issue usernames, which are
```

Comment:
@rzadp how do we keep the README in sync now? Does the developer need to remember to update it?

Reply:
There was a change for automatically generating a [simple readme](https://github.com/paritytech/polkadot-sdk/blob/d5346e72e54b040670b60475b90a3c467f03b08c/.github/scripts/generate-readmes.py#L28-L51), if there is no readme file at all.

If a readme file does exist, there's been no changes (not from me) - so the developer needs to remember to update, if that's how it was before. Some crates will want to keep handcrafted readmes.

Reply:
I updated the README.